docs/api/hazards.md

```diff
@@ -28,16 +28,29 @@ By "kidnap" we denote the action of manually lifting the robot and, eventually,
 The Create® 3 robot combines together different sensors data in order to determine when it's being kidnapped.
 A boolean status will be periodically published on the `kidnap_status` topic.
 
+### Reflective Infrared Sensors
 
-### The `ir_intensity` topic
+The Create 3 robot is equipped with 11 reflective IR sensors to detect hazards.
 
-Besides the aforementioned sensors, the Create® 3 robot is also equipped with [7 sets of IR emitters and receivers](/hw/mechanical/#ir-proximity-sensors) and it can use them to detect objects at close range.
+#### The `ir_intensity` topic
```

Comment:
This is a different level header than the ones that describe the hazard_detection and kidnap_status topics above as a result of having the Reflective Infrared Sensors header in line 31. Perhaps change the header for the hazard and kidnap to match?

Reply:
I thought about this, but I didn't want to break the hierarchy. Let me play with it.

Reply:
Looking at it, I think the extra heading I added is unnecessary; I'm just going to get rid of it and put everything on the same level.